### PR TITLE
Closes #560: Analysis page - chart date range selector

### LINKED ISSUE #560: Analysis page - chart date range selector

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js [hunk 1]
```diff
@@ -20,6 +20,90 @@ function extractIndex(element) {
...
+    
+    var monthStart = $('#' + monthStartId);
+    var yearStart = $('#' + yearStartId);
+    var monthEnd = $('#' + monthEndId);
+    var yearEnd = $('#' + yearEndId);
+    
+    
+    yearStart.change(function() {
+        
+        if (yearStart.val() > yearEnd.val()) {
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js]: Let's assume that start is 09-2001 and end is 02-2002. Then we change startYear to 2002.
This condition will fail and we will have start as 09-2002 which is later than end.
Am I right or missed something?

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js [hunk 2]
```diff
@@ -20,6 +20,90 @@ function extractIndex(element) {
...
+                
+            }
+        }
+        
+    });
+
+
+    yearEnd.change(function() {
+        
+        if (yearStart.val() > yearEnd.val()) {
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js]: Analogical to comment above

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js (hunk 1 above)
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js]: No, because I change in such a case the end month too (see the code below).

Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js]: Oh, sorry, you're right...

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js (hunk 2 above)
Comment (lukdumi) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/saos-common.js]: Right, thx

**Pull request closed without merging**